packages/astro/test/fixtures/astro-dynamic/src/pages/index.astro

```diff
@@ -7,6 +7,6 @@ import SvelteCounter from '../components/SvelteCounter.svelte';
 <body>
   <Counter client:load />
 
-  <SvelteCounter client:load />
+  <SvelteCounter:load />
```

Comment:
can you add a comment about this that says this is being left in to test backwards compat? That way someone won't accidentally change it.

Reply:
Yep! Good call